src/api/procedures/issueTokens.ts

```diff
@@ -9,6 +9,7 @@ import { bigNumberToBalance, portfolioToPortfolioKind, stringToTicker } from '~/
 export interface IssueTokensParams {
   amount: BigNumber;
   ticker: string;
+  portfolioId?: BigNumber;
```

Comment:
Do we have to worry about custodian portfolios? We might want to allow for a full portfolio ID here (`portfolio: {did: string; id: BigNumber}`).

You might need to check with core what the full intention is here.

Reply:
seems like its OK inferring the DID here